### DISCUSSION TITLE
Probelm adding backgrounds

### DISCUSSION THREAD

**Discussion by Flowersclouds** [Monika-After-Story/MonikaModDev#8536] [general - submod / external content]
Hello! I have an issue, Okay so, When I browse YT, I see these cool backgrounds And I decided to add them. I added the den , kitchen , space room ( 1- 3 ) , and garden view, But then I wanted to add more backgrounds So I attempted that, The problem is, That the rooms I wanted to add didn't have the same instructions as my last backgrounds, At first, Everytime I would add A new background It would crash my game, Not letting me see monika until I take the background out of my submods folder, I stopped trying after a while but now just started trying again, But now, Whenever I add a new background it wont show up when I launch my game and Go into locations, Can You please tell me what Im doing wrong?

**Comment by JinSakaiM** [comment]
you probably havent gotten the option to tell her about the classroom yet (Edit) - like in-game for example you can go to "Romance" when you press "Talk", but you need to unlock "Talk"-> "Hey... Monika"->"Location", because this is when you ask her about moving to a new place, so your not doing anything wrong, its just you havent unlocked the option to do it yet.
